Can't compile the project - forbidden dependencies

I just cloned the repository and tried to build it but I got an error:

```
javi@fooock intellij-solidity $ ./gradlew buildPlugin

Welcome to Gradle 6.5!

Here are the highlights of this release:
 - Experimental file-system watching
 - Improved version ordering
 - New samples

For more details see https://docs.gradle.org/6.5/release-notes.html

Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'intellij-solidity'.
> Failed to notify project evaluation listener.
   > Could not resolve all files for configuration ':compileOnly'.
      > Could not resolve org.jetbrains.intellij.deps.jflex:jflex:1.7.0-1.
        Required by:
            project :
         > Could not resolve org.jetbrains.intellij.deps.jflex:jflex:1.7.0-1.
            > Could not get resource 'https://jetbrains.bintray.com/intellij-third-party-dependencies/org/jetbrains/intellij/deps/jflex/jflex/1.7.0-1/jflex-1.7.0-1.pom'.
               > Could not GET 'https://jetbrains.bintray.com/intellij-third-party-dependencies/org/jetbrains/intellij/deps/jflex/jflex/1.7.0-1/jflex-1.7.0-1.pom'. Received status code 403 from server: Forbidden
   > Could not resolve all files for configuration ':compileOnly'.
      > Could not resolve org.jetbrains.intellij.deps.jflex:jflex:1.7.0-1.
        Required by:
            project :
         > Could not resolve org.jetbrains.intellij.deps.jflex:jflex:1.7.0-1.
            > Could not get resource 'https://jetbrains.bintray.com/intellij-third-party-dependencies/org/jetbrains/intellij/deps/jflex/jflex/1.7.0-1/jflex-1.7.0-1.pom'.
               > Could not GET 'https://jetbrains.bintray.com/intellij-third-party-dependencies/org/jetbrains/intellij/deps/jflex/jflex/1.7.0-1/jflex-1.7.0-1.pom'. Received status code 403 from server: Forbidden

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.5/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 8s
```

Any tip on how to proceed?

--- a/build.gradle
+++ b/build.gradle
@@ -59,6 +59,8 @@ intellij {
 
 repositories {
     jcenter()
+    maven { url "https://www.jetbrains.com/intellij-repository/releases" }
+    maven { url "https://cache-redirector.jetbrains.com/intellij-dependencies" }
 }
 
 configurations {

